### DISCUSSION TITLE
add payment with module template blazor web app

### DISCUSSION THREAD

**Discussion by khanhduatd186** [abpframework/abp#20590] [Q&A]
I want to add abp payment package to the project with template module, how do I do it? I can't find the documentation to do that, please help me, thank you.

**Comment by maliming** [comment]
hi

Please create a support ticket on https://support.abp.io/

Thanks.

**Reply by khanhduatd186** [reply]
hi maliming
https://abp.io/support/questions/7762/Issue-when-using-the-payment-module-with-the-template-module
